packages/db/src/db.generated.d.ts

```diff
@@ -186,8 +182,8 @@ export interface DB {
   attendee: Attendee;
   committee: Committee;
   company: Company;
-  "drizzle.DrizzleMigrations": DrizzleDrizzleMigrations;
```

Comment:
interesting, why was this table even here in the first place I wonder (we dont use drizzle)

Reply:
No idea! Weird